### DISCUSSION TITLE
更新包和启动报错和警告

### DISCUSSION THREAD

**Discussion by zybzbk** [vuepress-theme-hope/vuepress-theme-hope#4336] [Q&A] [answered]
在运行时，进入本地服务器会警告4 repetitive deprecation warnings omitted.
4 repetitive deprecation warnings omitted.
18 repetitive deprecation warnings omitted.
在更新包的时候会报错npm warn exec The following package was not found and will be installed: vp-update@2.0.0-rc.15

**Comment by Mister-Hope** [comment, marked as answer]
these are all.expected for now.
